client/src/components/FullProduct/FullProduct.js

```diff
@@ -1,38 +1,73 @@
-import React, { Component } from 'react';
+import React from 'react';
+import { useQuery, gql } from '@apollo/client';
+
 import MiniGallery from './MiniGallery';
 import ValueAttributes from './ValueAttributes';
 
-export class FullProduct extends Component {
-  _handleKeyUp = (event) => {
-    if (event.key === 'Escape') {
-      return this.props.onShowProduct(this.props.item);
+const GET_PRODUCT_BY_ID = gql`
```

Comment:
Lets move this into another file and all other gql string as well

Reply:
Sounds great! Thanks!
Done ✅👌